app/ts/library/utilities.ts

```diff
@@ -24,6 +24,14 @@ export function removeNonStringsAndTrim(...strings: (string | boolean | undefine
 	)
 }
 
+/**
+ * Generate a number within a number range
+ */
+
+export function getRandomNumberBetween(from: number, to: number): number {
```

Comment:
why do we need a random number generator?

Reply:
> Maybe bit too excessive to complain about the balance or every token? Could just leave it empty
> ![image](https://user-images.githubusercontent.com/13102010/222362216-697c8452-6c74-4da4-99e3-f753e8433cc3.png)

Moving the changeset to #84

Reply:
> why do we need a random number generator?

Just used in text with unknown length while loading to give it a "presumed" look